.pre-commit-config.yaml

```diff
@@ -4,3 +4,8 @@ repos:
     hooks:
       - id: black
         language_version: python3
+  - repo: https://github.com/pycqa/isort
+    rev: 5.8.0
+    hooks:
+      - id: isort
```

Comment:
As stated in the CLI example (https://pycqa.github.io/isort/docs/configuration/black_compatibility/) I think `isort` should run before black